### DISCUSSION TITLE
Feature Suggestion: Xbox One & Elite controller emulation

### DISCUSSION THREAD

**Discussion by pinecones-sx** [Valkirie/HandheldCompanion#717] [Ideas]
Please add Xbox One & Xbox One Elite controller emulation to allow mapping to the capture button & back paddle buttons.

**Comment by CasperH2O** [comment]
Controller emulation is done through the Vigem (https://github.com/ViGEm/ViGEmBus). Vigem is currently feature frozen and Vigem 2 is not yet released. 

If and when Vigem 2 comes out it will be investigated to update accordingly and support more simulated HID devices.
